specs/deposits.md

```diff
@@ -148,21 +155,25 @@ The contract has the following solidity interface, and can be interacted with ac
 
 #### L1 Attributes: Reference Implementation
 
+[l1-attrib-ref-implem]: #l1-attributes--reference-implementation
+
 A reference implementation of the L1 Attributes predeploy contract can be found in [L1Block.sol].
 
-[L1Block.sol]: /packages/contracts/contracts/L1Block.sol
+[L1Block.sol]: ../packages/contracts/contracts/L2/L1Block.sol
 
-The bytecode to add to the genesis file will be located in the `deployedBytecode` of the
-[JSON artifact file][l1-block-artifacts] corresponding to L1Block.sol.
+The bytecode to add to the genesis file will be located in the
+`deployedBytecode` of the [JSON artifact file][l1-block-artifacts] corresponding
+to L1Block.sol (this link will be broken unless you've built the contracts with
+`yarn build` in the `packages/contracts` directory).
```

Comment:
Maybe just remove this link and display the path textually instead. @maurelian wdyt?

Reply:
Agree. I made a suggestion below that I think gets this across well.